cmd/utils/flags.go

```diff
@@ -774,10 +778,10 @@ var (
 
 // MakeDataDir retrieves the currently requested data directory, terminating
 // if none (or the empty string) is specified. If the node is starting a testnet,
-// the a subdirectory of the specified datadir will be used.
+// then a subdirectory of the specified data-dir will be used.
```

Comment:
We use `datadir` throughout the code, we never use `data-dir`. Lets not introduce a new notation. Feel free to choose between `datadir` or `data directory` :)